deployer/src/main/java/org/wso2/msf4j/deployer/internal/DataHolder.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.msf4j.deployer.internal;
+
+import org.wso2.msf4j.MicroservicesRegistry;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * DataHolder for Swagger component.
+ */
```

Comment:
IMO we should add `@since <VERSION>` so that it is understandable by the user in which version this class is introduced from

Reply:
This is not a direct use by MSF4J end user. Therefore IMHO we don't need to add Since version

Reply:
This is mean for the developers and is a part of java doc conventions [1]

[1] http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@since